src/client/sandbox/node/element.js

```diff
@@ -532,12 +543,100 @@ export default class ElementSandbox extends SandboxBase {
         }, false);
     }
 
+    _refreshOnsubmitWrapper (form) {
+        var handler                 = null;
+        var storedOnsubmitAttrValue = nativeMethods.getAttribute.call(form, domProcessor.getStoredAttrName('onsubmit'));
+
+        /*eslint-disable no-new-func */
+        if (storedOnsubmitAttrValue)
+            handler = Function('event', storedOnsubmitAttrValue);
+        else
+            handler = form[ORIGIN_ONSUBMIT_PROPERTY_VALUE];
+        /*eslint-enable no-new-func */
+
+        this.setOnsubmit(form, handler);
+    }
+
+    _createSubmitButton () {
+        var button = nativeMethods.createElement.call(this.document, 'input');
+
+        button.type = 'submit';
+        this.listeners.initElementListening(button, ['click']);
+        this.listeners.addInternalEventListener(button, ['click'], stopPropagation);
+
+        return button;
+    }
+
+    _resubmitForm (form) {
+        var tempSubmitButton = this._createSubmitButton();
+
```

Comment:
Why do we need a temporary button.
As I know we can simple call `form.submit()`.